src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -557,6 +557,23 @@ describe('manage nsfs cli account flow', () => {
             expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InvalidArgumentType.message);
         });
 
+        it('cli unset new_buckets_path', async () => {
+            const { name } = defaults;
+            const account_options = { config_root, name, new_buckets_path: '\'\''};
```

Comment:
Why not use the `new_buckets_path` from defaults?
```suggestion
            const { name, new_buckets_path } = defaults;
            const account_options = { config_root, name, new_buckets_path};
```

Reply:
I want to update new_buckets_path as empty ('') to see that it unsets. new_buckets_path from defaults is a valid value